Project SF/Views/Tabs/Settings/SettingsView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SettingsView.swift
+//  Project SF
+//
+//  Created by Roman Esin on 11.07.2020.
+//
+
+import SwiftUI
+
+struct SettingsLabel<Destination: View>: View {
```

Comment:
Maybe rename to be more general (ie "NavigationLabel") and move to its own file

Reply:
Yep, just gotta fix some conflicts